### PR TITLE
Documentation for non-public cloud support for log analytics scaler.

### PR DESCRIPTION
**Pull request by v-shenoy**
Signed-off-by: Vighnesh Shenoy <vshenoy@microsoft.com>

Update log analytics scaler documentation.
https://github.com/kedacore/keda/issues/1916

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes #

### CODE REVIEW

File: content/docs/2.7/scalers/azure-app-insights.md [hunk 1]
```diff
@@ -46,8 +46,8 @@ for further details.
 - `activeDirectoryClientId` - Id of the Active Directory client. The client must have `Monitoring Reader` permissions for the Application Insights instance.
 - `activeDirectoryClientPassword` - Password of the Active Directory client password.
 - `metricFilter` - Further specify the metrics query using a filter. For example `cloud/roleName eq 'example`. (Optional)
-- `cloud` - Name of the cloud environment that the Event Hub belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
-- `appInsightsResourceURL` - Application Insights REST API URL of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).
+- `cloud` - Name of the cloud environment that the Application Insights belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
```
Comment (tomkerkhove) [inline content/docs/2.7/scalers/azure-app-insights.md:49]: Nit
```suggestion
- `cloud` - Name of the cloud environment that the Azure Application Insights instance belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
```

File: content/docs/2.7/scalers/azure-log-analytics.md [hunk 2]
```diff
@@ -59,6 +65,9 @@ triggers:
 - `query` - Log Analytics [kusto](https://docs.microsoft.com/en-us/azure/azure-monitor/log-query/get-started-queries) query, JSON escaped. You can use [this](https://www.freeformatter.com/json-escape.html) tool to convert your query from Log Analytics query editor to JSON escaped string, and then review YAML specific escapes.
 - `threshold` - Value that is used as a threshold to calculate # of pods for scale target.
 - `metricName` - Name to assign to the metric. (Optional, if not set KEDA will generate a name based on the workspaceId)
+- `cloud` - Name of the cloud environment that the Log Analytics workspace belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
```
Comment (tomkerkhove) [inline content/docs/2.7/scalers/azure-log-analytics.md:68]: Nit
```suggestion
- `cloud` - Name of the cloud environment that the Azure Log Analytics workspace belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
```

**Review by tomkerkhove — APPROVED** [review]
LGTM, added a few nits

File: content/docs/2.7/scalers/azure-log-analytics.md (hunk 2 above)
Comment (v-shenoy) [reply content/docs/2.7/scalers/azure-log-analytics.md]: Done.

File: content/docs/2.7/scalers/azure-app-insights.md [hunk 3]
```diff
@@ -46,8 +46,8 @@ for further details.
 - `activeDirectoryClientId` - Id of the Active Directory client. The client must have `Monitoring Reader` permissions for the Application Insights instance.
 - `activeDirectoryClientPassword` - Password of the Active Directory client password.
 - `metricFilter` - Further specify the metrics query using a filter. For example `cloud/roleName eq 'example`. (Optional)
-- `cloud` - Name of the cloud environment that the Event Hub belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
-- `appInsightsResourceURL` - Application Insights REST API URL of the cloud environment. (Required when `cloud` is set to `Private`, e.g. `https://login.chinacloudapi.cn/` for `AzureChinaCloud`).
+- `cloud` - Name of the cloud environment that the Azure Application Insights instance belongs to. (Values: `AzurePublicCloud`, `AzureUSGovernmentCloud`, `AzureChinaCloud`, `Private`, Default: `AzurePublicCloud`, Optional)
```
Comment (JorTurFer) [inline content/docs/2.7/scalers/azure-app-insights.md:49]: Is correct Azure Application Insights here or is an error?

**Review by JorTurFer — APPROVED** [review]

**Pull request merged**